elements/raviart-thomas.def

```diff
@@ -22,8 +22,9 @@ short-names:
   - RTcf (quadrilateral)
   - Ncf (hexahedron)
 complexes:
-  de-rham: P-,d-1,simplex
-  de-rham: Q-,d-1,tp
+  de-rham:
+  - P-,d-1,simplex
+  - Q-,d-1,tp
```

Comment:
looks like this change might fix one of the bullet points in https://github.com/DefElement/DefElement/issues/357